dask_ml/utils.py

```diff
@@ -23,8 +23,37 @@
 logger = logging.getLogger()
 
 
+def svd_flip_with_copy(u, v, u_based_decision=True):
```

Comment:
Rather than duplicating the entire implementation, can you copy the inputs and set them to not be readonly?